packages/atlas/src/views/viewer/EditMembershipView/EditMembershipView.tsx

```diff
@@ -53,6 +54,8 @@ export const EditMembershipView: React.FC = () => {
     }
   }, [activeMembership, activeMembershipLoading, resetForm])
 
+  const headTags = useHeadTags('Edit membership')
```

Comment:
Same question about the distinction between editing and adding

Reply:
Here you can only edit